New CI Post-Submit tests fail to find ci/pub_upgrade.sh

On this PR (just adding an icon):

https://github.com/openfoodfacts/smooth-app/pull/455

@github-actions
SmoothApp Post-Submit Tests / build (push) Failing after 1m — build

https://github.com/openfoodfacts/smooth-app/pull/455/checks?check_run_id=3146481565

```
Run ci/pub_upgrade.sh
  ci/pub_upgrade.sh
  shell: /bin/bash -e {0}
  env:
    JAVA_HOME_14.0.2_x64: /Users/runner/hostedtoolcache/jdk/14.0.2/x64
    JAVA_HOME: /Users/runner/hostedtoolcache/jdk/14.0.2/x64
    JAVA_HOME_14_0_2_X64: /Users/runner/hostedtoolcache/jdk/14.0.2/x64
    FLUTTER_ROOT: /Users/runner/hostedtoolcache/flutter/2.2.3-stable/x64
    PUB_CACHE: /Users/runner/hostedtoolcache/flutter/2.2.3-stable/x64/.pub-cache
/Users/runner/work/_temp/af7b9d55-5bb9-4d2f-9963-5346ab286843.sh: line 1: ci/pub_upgrade.sh: No such file or directory
Error: Process completed with exit code 1.
```

--- a/.github/workflows/postsubmit.yml
+++ b/.github/workflows/postsubmit.yml
@@ -4,12 +4,11 @@ on: push
     
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: ./packages/smooth_app
+        shell: bash
     steps:
-    
       - name: "Checkout code"
         uses: actions/checkout@v1
 
@@ -44,6 +43,7 @@ jobs:
       # Build apk.
       - name: "Build apk"
         run: flutter build apk --debug
+        working-directory: ./packages/smooth_app
       
       # Upload generated apk to the artifacts.
       - name: "Upload APK"

